profile.md

```diff
@@ -11,37 +11,144 @@ This section of the documentation contains things, that can't be described using
+:::
+::: tip
+If you don't define *k1* attribute, polynom3 will continue previous element at the starting point. 
+:::
+::: tip
+In many cases, you don't need to use mathematical equations of polynom3 and its derivatives. Usually, a simple adjustment of *x*, *y*, *k1* and *k2* attributes is enough.
+:::
+
+### Profile
+Parent element name: profile\
+Element name: profile
+
+Your defined profile can also contain build-in or your other predefined profiles.
+
+| Name  | Type   | Description                                                                         | Optional |
+| ----- | ------ | ----------------------------------------------------------------------------------- | -------- |
+| id    | String | The name of profile used                                                            |          |
+| x     | Float  | The end point of profile in position relative to the X-axis                         | True     |
+| y     | Float  | A profile shift in position relative to the Y-axis                                  | True     |
+| n     | Float  | A profile shift in position perpendicular (using normal vectors) to this profile    | True     |
+| dx    | Float  | A profile shift in position relative to the X-axis                                  | True     |
+| refx  | String | A reference to the X-axis, which X-point (from *x* attribute) is based on           | True     |
+
+::: warning
+Profile, by default, is shifted in position relative to Y-axis to match previously used element. *y* and *n* attibutes apply to this default shift. 
+:::
+
+::: tip
+The term "end point of profile" means the end point of use of given profile, not the end point of this profile in general. 
+:::
+
+### Arc
+Parent element name: profile\
+Element name: arc
+
+Arc element defines the arc that is part of a circle. It can be used e.g. to create elements that can't be created by beams. 
+
+| Name  | Type   | Description                                                                | Optional |
+| ----- | ------ | -------------------------------------------------------------------------- | -------- |
+| x     | Float  | The next point in position relative to the X-axis                          |          |
+| y     | Float  | The next point in position relative to the Y-axis                          |          |
+| x0    | Float  | The centre point of circle in position relative to X0-axis                 |          |
+| y0    | Float  | The centre point of circle in position relative to Y0-axis                 |          |
+| refx  | String | A reference to the X-axis, which profile is based on                       | True     |
+| refy  | String | A reference to the Y-axis, which profile is based on                       | True     |
+| refyx | Float  | A reference to X-point, which Y-point (from *y* attribute) is based on     | True     |
+
+::: warning 
+X0(Y0)-axis is the axis of global coordinates system. You can't change its reference.
```

Comment:
[x, y]0-axis are the axis of global coordinates system. You can't change its reference.